### DISCUSSION TITLE
Where are the classification images saved?

### DISCUSSION THREAD

**Discussion by zaqxs123456** [d8ahazard/sd_dreambooth_extension#385] [Q&A] [answered]
Sorry but I can't find where are the classification images saved, does it save those images on the drive or only inside the RAM? Thank you!

**Comment by D4ndellion** [comment, marked as answer]
the classification images are stored in the folder under `models/dreambooth/<yourmodel>/class_0` or similar

**Reply by zaqxs123456** [reply]
Found it! Thank you very much!
